docs/reference/esql/functions/auto_bucket.asciidoc

```diff
@@ -1,72 +1,118 @@
 [discrete]
 [[esql-auto_bucket]]
 === `AUTO_BUCKET`
-Creates human-friendly buckets and returns a `datetime` value for each row that
-corresponds to the resulting bucket the row falls into. Combine `AUTO_BUCKET`
-with <<esql-stats-by>> to create a date histogram.
 
-You provide a target number of buckets, a start date, and an end date, and it
-picks an appropriate bucket size to generate the target number of buckets or
-fewer. For example, this asks for at most 20 buckets over a whole year, which
-picks monthly buckets:
+*Syntax*
+
+[source,esql]
+----
+AUTO_BUCKET(field, buckets, from, to)
+----
+
+*Parameters*
+
+`field`::
+Date or numeric column from which to derive buckets.
```

Comment:
piconit: 
```suggestion
Numeric or date column from which to derive buckets.
```
-- just to be in sync with the order of types in the parameters sections.